app/react/portainer/environments/wizard/EnvironmentsCreationView/WizardKubernetes/KubeConfigTeaserForm.tsx

```diff
@@ -85,7 +86,8 @@ export function KubeConfigTeaserForm() {
                 isLoading={false}
                 disabled
               >
-                <i className="fa fa-plug" aria-hidden="true" /> Connect
+                <Icon icon="svg-plug" />
```

Comment:
use the icon prop of LoadingButton. and import the icon :)

Reply:
The loading button  wasn't diplaying the icon passed as a prop (see the LoadingButtonIcon funtion). I updated the loading button so that it can display icons now

Reply:
https://github.com/portainer/portainer/pull/7907/commits/9ba7bf0f2435274841a5c453146d343608eb36de